packit/api.py

```diff
@@ -252,6 +252,7 @@ def pkg_tool(self) -> str:
 
         return self.config.pkg_tool
 
+    # [TODO] Rename to something more reasonable
```

Comment:
would it require too many changes to change it right away?

Reply:
Should be just a regex replace, but I couldn't come up with some reasonable name for this :D maybe `common_env` and some nice docstring?

Reply:
I quite like `common_env`.

Reply:
+1

Reply:
`common_env` it is :D